core/storage/statistics/gae_models_test.py

```diff
@@ -368,6 +665,51 @@ def test_get_multi_stats_models(self) -> None:
         self.assertEqual(stat_models[2].exp_id, 'exp_id2')
         self.assertEqual(stat_models[2].exp_version, 1)
 
+    def test_get_multi_versions(self) -> None:
+        stats_models.ExplorationStatsModel.create(
+            'exp_id1', 1, 0, 0, 0, 0, 0, 0, {})
+        stats_models.ExplorationStatsModel.create(
+            'exp_id1', 2, 0, 0, 0, 0, 0, 0, {})
+
+        stat_models = stats_models.ExplorationStatsModel.get_multi_versions(
+            'exp_id1', [1, 2]
+        )
+
+        assert stat_models[0] is not None
+        assert stat_models[1] is not None
```

Comment:
I think you can do `self.assertIsNotNone()`? Ditto below

Reply:
Adding `self.assertIsNotNone()` in place of `assert stat_model is not None` results in failing mypy checks with the error log reading `Item "None" of "Optional[ExplorationStatsModel]" has no attribute "id"  [union-attr]` . The syntax of using `assert model is not None` was used previously in other tests to avoid the same errors.

Reply:
Ahh okay, that sounds fine